resotoclient/http_client/aiohttp_client.py

```diff
@@ -0,0 +1,215 @@
+from resotoclient.http_client import AsyncHttpClient
+from resotoclient.http_client import HttpResponse
+from typing import Dict, Optional, Callable, Union, AsyncIterator, Awaitable
+from resotoclient.models import JsValue
+from resotoclient.jwt_utils import encode_jwt_to_headers
+import aiohttp
+import ssl
+from yarl import URL
```

Comment:
Do we need an external dependency for URL parsing? Python comes with urllib.parse which provides split and unsplit functions.

Reply:
It comes as a part of the AIOhttp library, so we get it for free.